pkgs/tools/misc/inav-blackbox-tools/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "inav-blackbox-tools";
+  version = "2021-04-22";
+
+  src = fetchFromGitHub {
+    owner = "iNavFlight";
+    repo = "blackbox-tools";
+    rev = "0109e2fb9b44d593e60bca4cef4098d83c55c373";
```

Comment:
Any particular reason for this rev vs. taking the last stable v0.4.5?

Reply:
There's only been one commit since then, but it's an important bug fix (for me)! Given the very low rate of development there I'm not sure when to expect a new release.